plugins/destiny.py

```diff
@@ -343,6 +343,35 @@ def triumph(text, nick, bot):
 
 @hook.command('xur')
 def xur(text, bot):
+    # reset happens at 9am UTC, so subtract that to simplify the math
+    now = datetime.datetime.utcnow() - datetime.timedelta(hours=9)
+
+    # xur is available from friday's reset until sunday's reset, i.e. friday (4) and saturday (5)
+    if now.weekday() not in [4, 5]:
+        del CACHE['xur'] # data will be wrong anyway
```

Comment:
Absolutely not. this breaks the "last" functionality. Also has not significant benefit if a "last" option wasn't made.